src/getting-started/appdev-process.md

```diff
@@ -1,47 +1,141 @@
 
-### Barファイルのビルド
+The development of the application is done for the data subject Cell part and the application Cell part.
 
+## Box and bar file installation
 
-### 開発用データ主体CellへのBarインストール
+The application needs to take the same data structure on Box in order to perform the intended behavior regardless of which data subject Cell is used. In Personium, before the user of the application uses the application, Box installation is performed using the bar file that defines the data structure of the data subject Cell, and the application-specific data structure is built on Box. Please refer to the following documents for details.
 
+* [Box installation](../apiref/007_Box_install.md)
+* [bar file](../apiref/301_Bar_File.md)
 
-### Box上の開発
+The development of the data-oriented Cell part is the construction of the data structure on Box and the output of the bar file.
 
+## Template app
 
-### アプリCell上の開発
+The Personium community offers templates and deployment tools for [React](https://reactjs.org/) based JavaScript apps.
+
+[personium-blank-app](https://github.com/personium/personium-blank-app)
+
+By using this tool, you can develop the original application by customizing from the minimum data subject Cell and application Cell. It also simplifies development work such as building bar files and uploading them to the Cell application.
+
+### Flow of initial construction
+
+The flow of initial construction of the template application is as follows.
+
+1. clone of personium-blank-app to local development environment
+2. Construction of application Cell
+   1. Editing the configuration file
+   2. Deploy to application Cell (execute `npm run deploy`)
+   3. ACL setting of file on application Cell
+3. Construction of data subject Cell
+   1. Build the bar file (run `npm build-bar`)
+   2. Box installation in the development data-based Cell
+
+Please refer to [personium-blank-app](https://github.com/personium/personium-blank-app) for the detailed procedure.
+
+> [Unit Manager explained in the previous section](./appdev-management-tool.md) can be used to perform operations on the Cell such as ACL setting and Box installation.
+
+## Application development flow
+
+After building the template application, you can develop by repeating the following.
+
+* Development on the development data-based Cell
+  1. Creation of data structure on Box by Unit Manager
+  2. App operation check
+  3. Output of bar file by Unit Manager
+  4. Commit the bar file to the code repository
+* Development on App Cell
+  1. File editing in local development environment (HTML/JavaScript/CSS etc.)
+  2. Deploy to application Cell (execute `npm run deploy`)
+  3. ACL setting of file on application Cell
+  4. App operation check
+  5. Commit the files on the application Cell to the code repository
+
+> Instead of deploying to the application Cell, you can start the development Web server on the local development environment and check the operation. In that case, run `npm run debug`.
```

Comment:
```suggestion
> Instead of deploying to the app Cell, you can start the development Web server on the local development environment and verify the functions. In that case, run `npm run debug`.
```